source/app-analysis/run-analysis.txt

```diff
@@ -0,0 +1,80 @@
+.. _rm-run-app-analysis:
+
+==========================
+Run Pre-Migration Analysis
+==========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can run Pre-Migration Analysis against a connected relational database to
+review potential migration risks before migrating data to MongoDB. Running
+Pre-Migration Analysis generates a report that identifies risks and provides
+guidance on how to mitigate them.
+
+About this Task
+---------------
+
+.. important::
+
+   Relational Migrator supports Pre-Migration Analysis for Microsoft SQL
+   Server, MySQL, Oracle, and PostgreSQL. Pre-Migration Analysis isn't
+   supported for databases connected through a JDBC driver. For more
+   information on supported databases and versions, see 
+   :ref:`supported-databases`.
+
+Before you Begin
+----------------
+
+- :ref:`Create a project with a live database connection 
+  <rm-create-project-live>`
+
+Steps
+-----
+
+.. procedure::
+   :style:  normal
+
+   .. step:: Open your project
+
+      If your project has no Pre-Migration Analysis reports, Relational
+      Migrator displays help text to suggest analyzing your application.
+
+   .. step:: Navigate to the :guilabel:`Pre-Migration Analysis` tab
+
+      You can review previous reports here, or run Pre-Migration Analysis.
+
+   .. step:: Select a relational database connection if one isn't already set
+      
+      In the Database Connections bar, select the saved connection from the :guilabel:`Relational DB` drop-down.
+
+   .. step:: Click :guilabel:`Run analysis`
+
+      If you didn't save credentials when saving the relational database
+      connection, enter them now and then click :guilabel:`Connect
+      and run analysis` to continue.
+      
+      Relational Migrator analyzes the connected source database. Depending on
+      the size of the database, this can take (30 minutes to an hour?). You can
+      click :guilabel:`Terminate` if you need to stop this process for any
+      reason.
+
+      Once complete, Relational Migrator shows the resulting report.
+
+      .. important::
+
+         If Relational Migrator encounters errors, it displays the message "An
+         error occurred during the analysis. As a result, there may be
+         information missing from the report.". Review (logs?) for more 
```

Comment:
If an error occurred during analysis, where can they go to find more detailed info?